pkgs/by-name/di/dioxus-cli/package.nix

```
@@ -33,11 +36,27 @@ rustPlatform.buildRustPackage rec {
 
   OPENSSL_NO_VENDOR = 1;
 
+  # wasm-opt-sys build.rs tries to verify C++17 support, but the check appears to be faulty.
+  postPatch = ''
+    pushd $cargoDepsCopy/wasm-opt-sys
```

I think we either have to patch ourselves this release or remove the optimizations buildFeature
```suggestion
  postPatch = ''
    substituteInPlace src/build/bundle.rs \
      --replace-fail 'self.run_wasm_opt(&self.build.exe_dir())?;' 'self.run_wasm_opt(&self.build.wasm_bindgen_out_dir())?;'
      
    pushd $cargoDepsCopy/wasm-opt-sys
```